### PR TITLE
Updated help and community links.

### LINKED ISSUE #1123: Add discord reference in Django community page
It can be useful to add the URL to join the official Django discord server to the Django Community page:
https://www.djangoproject.com/community/

### PR DESCRIPTION
**Pull request by crydotsnake**
This pull request fixes #1123

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,4 @@ djangoproject/static/js/lib/unveil/jquery-1.9.1.min.js
 djangoproject/static/js/lib/unveil/index.html
 djangoproject/static/js/lib/unveil/img
 node_modules/
+.idea
```
Comment (carltongibson) [inline .gitignore:17]: Can I ask you to drop this change, preferring a local `.git/info/exclude`. Thanks.

**Review by carltongibson — COMMENTED** [review]
Hi @crydotsnake — Thanks for picking this up. 

IIUC, the main target here is the top of the commity page here: https://www.djangoproject.com/community/

(So Forum, Discord, then Mailing lists.) 

Also though there are a few different sidebar link sections, some of which could have the forum added too. (They look like they've drifted apart slightly over time... 🤔 ) — Would you be up for a slightly bigger audit, and re-unification? (Not a big project, but whilst we've got it open...) 

Let me know. 🙂

**Comment by crydotsnake** [comment]
> Also, if you git pull --rebase on main you can get rid of the merge commit. The README changes could be combined and we can pull those in no-problem. Thanks!

Hello @carltongibson,

My fork repository is already up to date.

**Comment by carltongibson** [comment]
Hi @crydotsnake. 

I specifically mean https://github.com/django/djangoproject.com/pull/1160/commits/3ed902b1088084c6218a16f0e2012c474584bb13 — which is a merge commit. If you are able to rebase that will disappear. 

Don't worry if not, I can rebase on merging. More significant is whether you can take on looking at the other points?

**Comment by crydotsnake** [comment]
Hi @carltongibson 

I try to rebase but its already on the latest changes.

> Don't worry if not, I can rebase on merging. More significant is whether you can take on looking at the other points?

Yes i will!

**Comment by carltongibson** [comment]
Super thanks! 🏅 

(Don't worry about the git changes: If you push your edits I can adjust when merging. Thanks!)

File: .gitignore (hunk 1 above)
Comment (crydotsnake) [reply .gitignore:17]: Thanks for the hint! 🙂

**Comment by carltongibson** [comment]
Well homepage: 
Screenshot 2022-04-19 at 15 39 07

And then sidebar: 

Screenshot 2022-04-19 at 15 39 07

But if I click the link there to the community page, this is at the bottom: 

Screenshot 2022-04-19 at 15 39 07

And that's not the only sidebar going through the site. 

So I guess it's, have a little browse and audit which are the different sidebar blocks, and can we make them more uniform? 

e.g. This on the docs: 

Screenshot 2022-04-19 at 15 42 23

And so on. 

I think an overview — an audit — would clear it up in not too long™ — there are only a few variations, but it looks like they've drifted over time. 

Make sense?

**Comment by crydotsnake** [comment]
What would think about adding the Forum and Discord Link also to the "Get Involved" Section in the footer?


Bildschirmfoto 2022-04-19 um 15 53 05

**Comment by carltongibson** [comment]
Sorry, I realise I dragged in the same screen shot multiple times above. 🤦 

> What would think about adding the Forum and Discord Link also to the "Get Involved" Section in the footer?

Yes — either that or linking to one central place (the community page?) where all the options are correctly there. The key point (I think) is that it's quite messy as is, because it's not the same everywhere, so spotting the _Wheres_ and making them the _Same_ would go a long way (I think, having you made me look at it, by taking this on 🙂)

**Comment by crydotsnake** [comment]
> But if I click the link there to the community page, this is at the bottom:

Then I would recommend moving those elements:

Bildschirmfoto 2022-04-19 um 16 01 26

To the "Get Help?" section. In my opinion, this makes more sense. What do you think?

**Comment by crydotsnake** [comment]
And what would you think, if we move the "Get Help" Section under the "Get Involved" Section:
Bildschirmfoto 2022-04-19 um 16 14 36

**Comment by carltongibson** [comment]
@crydotsnake Sounds likely... — I guess I'd say to come up with your unified suggestion(s) and then we can agree. 
(It's hard to tell from the screenshots exactly what's where, but I'm pretty sure your judgement will be on-track 🙂)

**Comment by crydotsnake** [comment]
> @crydotsnake Sounds likely... — I guess I'd say to come up with your unified suggestion(s) and then we can agree. (It's hard to tell from the screenshots exactly what's where, but I'm pretty sure your judgement will be on-track 🙂)

I pushed my changes. Let me know what you think! :)

File: djangoproject/templates/base_community.html [hunk 2]
```diff
@@ -17,13 +17,23 @@ <h1 class="visuallyhidden">Additional Information</h1>
...
-    <dd>chat with other Django users</dd>
     <dt><a href="https://code.djangoproject.com/">Ticket system</a></dt>
     <dd>report bugs and make feature requests</dd>
     <dt><a href="https://dashboard.djangoproject.com/">Development dashboard</a></dt>
     <dd>see what's currently being worked on</dd>
   </dl>
+  <h2>Get Help</h2>
+  <dl class="list-links">
+    <dt><a href="{% url 'document-detail' lang='en' version='stable' url='faq' host 'docs' %}">Check our FAQ first.</a></dt>
+    <dd>If you have a basic question that's not answered by the FAQ, file a ticket to tell us you think it should be in there.</dd>
```
Comment (smithdc1) [inline djangoproject/templates/base_community.html:28]: As we're here. 🙂

To me, this reads as " open a ticket on trac to get support". I'm not sure that's quite what's intended? 🤔

Maybe reword to something that describes the type of things in the faq?

Comment (crydotsnake) [reply djangoproject/templates/base_community.html:28]: Hello @smithdc1, nice to see you :wave:

Good idea!
Maybe something like:

"In our FAQ, you will find a list with the basic question and answers about the Django Open-Source Project.

Comment (crydotsnake) [reply djangoproject/templates/base_community.html:28]: Or as an alternative:

"In our FAQ you will find a list of the most important questions about the Django open source project for getting started."

Comment (carltongibson) [reply djangoproject/templates/base_community.html]: I went with the line from from the docs template: `Try the FAQ — it's got answers to many common questions.` — Thanks both! 🏅

**Review by carltongibson — APPROVED** [review]
Thanks @crydotsnake! I adjusted the commits slightly, and pushed small cosmetic edits. This is a nice improvement. 

(It'll go live on the next deploy.) 

Welcome aboard! ⛵

**Comment by crydotsnake** [comment]
Thanks a lot!! @carltongibson 🤩
